docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to Devworkspace-Operator
+
+Hello there! Thank you for choosing to the contributing to devfile/devworkspace-operator. Navigate through the following to understand more about contributing here.
+
+- [Contributing to Devworkspace-Operator](#contributing-to-devworkspace-operator)
+- [How to Contribute](#how-to-contribute)
+  - [Set up your Development Environment](#set-up-your-development-environment)
+      - [Running the controller locally.](#running-the-controller-locally)
+  - [Testing Changes](#testing-changes)
+      - [Test run the controller](#test-run-the-controller)
+      - [Developing Webhooks](#developing-webhooks)
+  - [Signing-off on Commits](#signing-off-on-commits)
+
+# How to Contribute
+To contribute to the devworkspace-operator project, developers should follow the fork and pull request workflow. 
+## Set up your Development Environment
+
+
+**1.** Fork [Devworkspace operator](https://github.com/devfile/devworkspace-operator) repository.
+
+**2.** Clone your forked copy of the project.
+
+```
+git clone https://github.com/<your-github-username>/devworkspace-operator.git
+```
+
+#### Running the controller locally.
+
+In the steps listed below, we set up the development environment using a minikube cluster. 
```

Comment:
Might be useful to link https://minikube.sigs.k8s.io/docs/start/ for "minikube cluster". The link includes installation instructions.